models/intermediate/user_stats.sql

```diff
@@ -0,0 +1,117 @@
+WITH combined_events_initiator AS (
+    SELECT
+        'shot' AS event_type,
+        s.initiator_user_id,
+        COALESCE(s.initiator_type, 'unknown') AS initiator_type,
```

Comment:
We should move these kinds of column modifications to the staging files for consistency in future